src/intl/translated-messages.json

```diff
@@ -639,10 +638,8 @@
     "utilizationCardDiskUsed": "{used} von {total} {storageUnits} verwendet",
     "utilizationCardLegendAvailable": "Verfügbar",
     "utilizationCardLegendAvailableP": "% verfügbar",
-    "utilizationCardLegendAvailableWithDetails": "Verfügbar {value}% ({diskPath})",
```

Comment:
Please re-apply https://github.com/oVirt/ovirt-web-ui/pull/1541
Translations for 2 labels were manually fixed there:
1. `utilizationCardLegendAvailableWithDetails`
2. `utilizationCardLegendUsedWithDetails`

Reply:
Fixed in Zanata

Reply:
I'll do another pull and update the PR.  :smile: